modules/tsb/cp/manifests/tctl/tctl-controlplane.sh.tmpl

```diff
@@ -7,7 +7,13 @@
 /usr/bin/env tctl x gitops grant ${cluster_name}
 
 # Allow managing platform Roles with the cluster service account
-/usr/bin/env tctl apply -f - <<EOF
+tctl get ab admin/rbac -o json >/tmp/rbac.json
```

Comment:
I would keep in the same directory as the rest of outputs for the existing script, makes troubleshooting or manual intervention easier if required

```
tsbadmin@r153d6-1-jumpbox:~$ pwd
/home/tsbadmin
tsbadmin@r153d6-1-jumpbox:~$ ls
gke-r153d6-us-east1-1-cluster.yaml  gke-r153d6-us-east1-1-service-account.jwk  gke-r153d6-us-east1-1-tctl-controlplane.sh
```

I would still keep ```/usr/bin/env``` in front of tctl and jq for the portability sake...

Reply:
Done.